policy/lib/bundles.rego

```diff
@@ -93,4 +93,7 @@ bundle(task) := refs.task_ref(task).bundle
 _collection(ref) := items {
 	full_collection := data["task-bundles"][ref.repo]
 	items := time_lib.acceptable_items(full_collection)
+} else := items {
+	# In case data["task-bundles"][ref.repo] doesn't exist
+	items := []
 }
```

Comment:
Could write it like this, wdyt? I thought the longer one was more readable.. :shrug: .

```suggestion
} else := [] # In case data["task-bundles"][ref.repo] doesn't exist
```

Reply:
Not really important... I prefer the shorter version